[pytorch] change op dependency output to use double-quoted strings

ljk53 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32464 [pytorch] change op dependency output to use double-quoted strings**

Summary:
Changed to double quoted strings to make FB linter happy.

Differential Revision: [D19507859](https://our.internmc.facebook.com/intern/diff/D19507859)

ljk53 committed:
[pytorch] change op dependency output to use double-quoted strings

Summary:
Changed to double quoted strings to make FB linter happy.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] change op dependency output to use double-quoted strings"


Summary:
Changed to double quoted strings to make FB linter happy.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] change op dependency output to use double-quoted strings"


Summary:
Changed to double quoted strings to make FB linter happy.

Differential Revision: [D19507859](https://our.internmc.facebook.com/intern/diff/D19507859)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 53e0352f:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=14e0bec9f2153d88cfa05d798fced87dc28873e4&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 14e0bec9 since Jan 22
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  Since your merge base is older than `viable/strict`, run these commands:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=53e0352f5c7b0e1658f788112f12d3a1d5d9d0d3).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=89700397) from Jan 22 until Jan 22



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ljk53 committed:
Update on "[pytorch] change op dependency output to use double-quoted strings"


Summary:
Changed to double quoted strings to make FB linter happy.

Differential Revision: [D19507859](https://our.internmc.facebook.com/intern/diff/D19507859)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] change op dependency output to use double-quoted strings"


Summary:
Changed to double quoted strings to make FB linter happy.

Differential Revision: [D19507859](https://our.internmc.facebook.com/intern/diff/D19507859)

[ghstack-poisoned]
